compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -265,6 +266,16 @@ def start_endpoint(
                 )
                 Endpoint.pidfile_cleanup(endpoint_dir)
 
+        if endpoint_config.endpoint_setup:
+            try:
+                subprocess.check_call(endpoint_config.endpoint_setup, shell=True)
+            except Exception:
```

Comment:
I don't mind `check_call` (from our conversation) per se, but from a debugging perspective, I'm wondering if we can be more helpful.  I'm seeing "something isn't set up as my `endpoint_setup` script should have made it, but it didn't fail.  So ... what _did_ it do?"  Does this capture the output to the endpoint logs already?

Reply:
Good point - no, this doesn't capture logs, but I agree that it should. At first glance, this is a [non-trivial process](https://codereview.stackexchange.com/q/6567/235544), so I'll open a follow-up PR to avoid muddling up this one.